### PR TITLE
Feature/init project setup

### LINKED ISSUE #20: Initial project setup - first commit (.gitignore, libs dependencies, project structure)

### PR DESCRIPTION
**Pull request by nbogdand**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">PetOwner</string>
```
Comment (andreimesina) [inline app/src/main/res/values/strings.xml]: Should we add a space between _Pet_ and _Owner_ to improve readability?

File: app/build.gradle [hunk 2]
```diff
@@ -1,7 +1,7 @@
-plugins {
-    id 'com.android.application'
-    id 'kotlin-android'
-}
+apply plugin: 'com.android.application'
```
Comment (andreimesina) [inline app/build.gradle]: Can't we still use the `plugins` structure? I believe the `apply plugin: ...` syntax is going to be deprecated soon

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (nbogdand) [reply app/src/main/res/values/strings.xml]: Changed

File: app/build.gradle (hunk 2 above)
Comment (nbogdand) [reply app/build.gradle]: Changed

**Review by andreimesina — APPROVED** [review]

**Pull request merged**